h2o-core/src/main/java/water/server/ServletUtils.java

```diff
@@ -173,6 +175,21 @@ public static String getDecodedUri(HttpServletRequest request) {
       throw new RuntimeException(e);
     }
   }
+  
+  public static String[] parseUriParams(String uri, HttpServletResponse response, Pattern p, int numParams) throws IOException {
```

Comment:
Minor: There is only one pattern really right now, the method is static anyway and can internally refer to the `Pattern p` instead of passing it as an additional parameter. A matter of preference really.